tests_practice/my_func.py

```diff
@@ -0,0 +1,134 @@
+import math
+import string
+
+
+def func_1(a, b):
```

Comment:
documenting your code, and name your functions like what they do

Reply:
Done!